encoding/json/_stringify_test.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+import { assertEquals, assertRejects } from "../../testing/asserts.ts";
+import {
+  ConcatenatedJSONStringifyStream,
+  JSONLinesStringifyStream,
+  StringifyStreamOptions,
+} from "./_stringify.ts";
+
+async function assertValidStringify(
+  transform:
+    | typeof ConcatenatedJSONStringifyStream
+    | typeof JSONLinesStringifyStream,
+  chunks: unknown[],
+  expect: string[],
+  options?: StringifyStreamOptions,
+) {
+  const r = new ReadableStream({
+    start(controller) {
+      for (const chunk of chunks) {
+        controller.enqueue(chunk);
+      }
+      controller.close();
+    },
+  });
```

Comment:
nitpick: any reason this doesnt use `readableStreamFromIterable`?

Reply:
Thank you for pointing this out.